src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/10 Page/keyboard.md

```diff
@@ -0,0 +1,24 @@
+---
+title: 'keyboard()'
+excerpt: 'Browser module: page.keyboard method'
+---
+
+Returns the [Keyboard](javascript-api/k6-experimental/browser/keyboard/) instance to interact with a virtual keyboard on the page.
+
```

Comment:
We should probably have a _Returns_ sub heading in here.

Reply:
Resolve in https://github.com/grafana/k6-docs/commit/95398b184f9afe52df89593693d16dfa2d7085c5